stdlib/distutils/log.pyi

```diff
@@ -8,18 +8,18 @@ FATAL: Final = 5
 
 class Log:
     def __init__(self, threshold: int = 3) -> None: ...
-    def log(self, level: int, msg: str, *args: Any) -> None: ...
-    def debug(self, msg: str, *args: Any) -> None: ...
-    def info(self, msg: str, *args: Any) -> None: ...
-    def warn(self, msg: str, *args: Any) -> None: ...
-    def error(self, msg: str, *args: Any) -> None: ...
-    def fatal(self, msg: str, *args: Any) -> None: ...
+    def log(self, level: int, msg: str, *args: object) -> None: ...
+    def debug(self, msg: str, *args: object) -> None: ...
+    def info(self, msg: str, *args: object) -> None: ...
+    def warn(self, msg: str, *args: object) -> None: ...
+    def error(self, msg: str, *args: object) -> None: ...
+    def fatal(self, msg: str, *args: object) -> None: ...
 
-def log(level: int, msg: str, *args: Any) -> None: ...
-def debug(msg: str, *args: Any) -> None: ...
-def info(msg: str, *args: Any) -> None: ...
-def warn(msg: str, *args: Any) -> None: ...
-def error(msg: str, *args: Any) -> None: ...
-def fatal(msg: str, *args: Any) -> None: ...
+def log(level: int, msg: str, *args: object) -> None: ...
+def debug(msg: str, *args: object) -> None: ...
+def info(msg: str, *args: object) -> None: ...
+def warn(msg: str, *args: object) -> None: ...
+def error(msg: str, *args: object) -> None: ...
+def fatal(msg: str, *args: object) -> None: ...
```

Comment:
Similar here. While `args` accepts anything if `%s` is used as format string, the same isn't true for other format strings:

```python
>>> import logging
>>> logging.warn("%i", "")
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.11/logging/__init__.py", line 1110, in emit
    msg = self.format(record)
          ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 953, in format
    return fmt.format(record)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 687, in format
    record.message = record.getMessage()
                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/logging/__init__.py", line 377, in getMessage
    msg = msg % self.args
          ~~~~^~~~~~~~~~~
TypeError: %i format: a real number is required, not str
Call stack:
  File "<stdin>", line 1, in <module>
Message: '%i'
Arguments: ('',)
```

Reply:
Ah right, depends on the format